app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -18,56 +19,74 @@ public function __construct()
         $this->middleware('role:admin,staff', ['only' => ['store', 'update', 'destroy']]);
     }
 
+    /**
+     * Show overview of campaigns and their IDs.
+     */
+    public function index()
+    {
+        $campaigns = Campaign::all();
+
+        // Format start and end dates how we want them to be viewed.
+        foreach ($campaigns as $campaign) {
+            $campaign = $this->formatStartEndDates($campaign, '');
```

Comment:
This should be done in the template (no need to iterate over and modify the models in-memory here).